spec/factories/miq_group.rb

```diff
@@ -1,7 +1,8 @@
 FactoryGirl.define do
   factory :miq_group do
+    guid { MiqUUID.new_guid }
+
     sequence(:description) { |n| "Test Group #{seq_padded_for_sorting(n)}" }
-    sequence(:guid) { |n| UUIDTools::UUID.parse_int((0x87654321 << 96) + n) }
```

Comment:
Interesting...we did something similar with IP addresses and MAC addresses for other factories [here](https://github.com/ManageIQ/manageiq/blob/master/spec/support/factory_girl_helper.rb#L5-L14).  I wouldn't mind having a guid_from_seq method there.